package.json

```diff
@@ -31,7 +31,7 @@
     "co-request": "^1.0.0",
     "deep-diff": "^0.3.0",
     "five-bells-condition": "~2.0.0",
-    "five-bells-shared": "~9.1.2",
+    "five-bells-shared": "~9.2.1",
```

Comment:
This was also updated here: https://github.com/interledger/five-bells-ledger/commit/d682b3c2480a981939f46db0afb06218251a8d7e, which is probably causing the merge conflicts.